### ISSUE TITLE
Accessibility Features verification

### ISSUE THREAD

**Issue by ludtkemorgan** [bloom-housing/bloom#4440]
The Accessibility Features section is already in the core codebase but it hasn't been tested in a while.

Verify that the partner section appears when the `enableAccessibilityFeatures` is set to true for the jurisdiction.
Verify that when editing and creating listing the values are properly stored
Verify that the accessibility features section appears on public when a listing has saved values

![image](https://github.com/user-attachments/assets/9bac8870-4d04-4ed1-94c5-b10b0200a5ee)

**Comment by mcgarrye** [comment]
Add backend unit tests and frontend integration tests

**Comment by ludtkemorgan** [comment]
QA notes:
- This has been enabled in core https://partners.bloom.exygy.dev/ for the Bloomington jurisdiction
- Go to [add listing page](https://partners.bloom.exygy.dev/listings/add) and select Bloomington jurisdiction
  - The "Accessibility Features" section should be displayed
  - You should be able to save the listing after selecting accessibility features and is should save correctly (should display on listing detail page on partners after save)
  - The "Accessibility Features" section should not be displayed if you select a different jurisdiction and save with that jurisdiction
- Go to listing edit page and the same functionality should exist

**Comment by sarahlazarich** [comment]
@ludtkemorgan this looks good and is responsive to the changing jurisdiction! 

I have the same question here about the listings export, but I think that may be a follow-up?

**Comment by ludtkemorgan** [comment]
@sarahlazarich shoot also didn't realize this was part of the export. I'll create a follow-up ticket that will have both of them added to the CSV.

**Comment by sarahlazarich** [comment]
Handling CSV in a follow-up, good to release
